pvlib/tests/iotools/test_psm3.py

```diff
@@ -50,47 +65,47 @@ def assert_psm3_equal(header, data, expected):
 
 
 @needs_pandas_0_22
```

Comment:
can you remind me why all of these tests need pandas 0.22?

Reply:
I was hoping you could tell me =)

Reply:
I'll try taking them off locally and see what errors pop out.

Reply:
@CameronTStark see [this comment](../694#issuecomment-487272722) in #694
>Bummer, I think the `pandas.to_datetime()` API changed after pandas-0.16 so the python-27-min build fails. I'll try to decorate test_psm3 with `@needs_pandas_0_22`

**So** since we've dropped Python-2.7 support, we just need to check if it's still valid for Python-3.5, otherwise we can remove these.

**Also** since we bumped pandas to v0.18.1 in [pvlib-0.7.0](https://pvlib-python.readthedocs.io/en/stable/whatsnew.html#v0-7-0-december-18-2019) we can probably definitely drop these now.

Reply:
Yep, the ability to assemble a datetimes from a sequence `(year, month, ...)` was new in [pandas-0.18](https://pandas.pydata.org/pandas-docs/version/0.18/whatsnew.html#whatsnew-0181-enhancements-assembling)

Reply:
Thanks for researching this. I took off the decorators locally to run the tests and didn't have an issue with my pandas==1.0.1 environment. I tried setting up an environment with 0.18.1 and couldn't get it to install so I can't confirm for that scenario. 

Like the `pytest-remotedata` I think this should be addressed in another issue so we can get this across the finishline.